spec/controllers/hq/admin_users_controller_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe Hq::AdminUsersController, type: :controller do
   let(:admin_users) { build_stubbed_list :admin_user, 2 }
+  let(:admin_user) { admin_users.first }
```

Comment:
Yes I wondered about which to use, and in the end went for the one that was slightly more efficient. But reconsidering it I think you're right about it being cleaner to use a separate build_stubbed, so I've changed that.